src/gui/folderman.cpp

```diff
@@ -849,7 +849,7 @@ Folder *FolderMan::folderForPath(const QString &path)
     foreach(Folder* folder, this->map().values()) {
         const QString folderPath = folder->cleanPath()+QLatin1Char('/');
 
-        if(absolutePath.startsWith(folderPath)) {
+        if(absolutePath.startsWith(folderPath, Qt::CaseInsensitive)) {
```

Comment:
Utility::isMac() should also generally be casesensitive, so maybe `Utility::isWindows() ? Qt::CaseInsensitive : Qt::CaseSensitive`?